login/meson.build

```diff
@@ -72,19 +73,21 @@ if get_option('pam-glome')
         install : true,
         install_dir : join_paths(get_option('libdir'), 'security'))
 
-    libpamtest = dependency('libpamtest', required : false)
-    if libpamtest.found()
-        pam_test = executable(
-            'pam_test', 'pam_test.c',
-            dependencies : [libpamtest])
-        custom_target('pam_service',
-            build_by_default : true, output : [ 'pam_service' ],
-            command : [ 'mkdir', '@OUTPUT@' ])
-        test('pam test', pam_test,
-            env: [ 'LD_PRELOAD=libpam_wrapper.so', 'PAM_WRAPPER=1',
-                'PAM_WRAPPER_SERVICE_DIR=' +
-                join_paths(meson.build_root(), 'login', 'pam_service'),
-                'PAM_GLOME=' +
-                join_paths(meson.build_root(), 'login', 'pam_glome.so') ])
+    if get_option('tests')
+        libpamtest = dependency('libpamtest', required : false)
```

Comment:
Should we require libpamtest when PAM is wanted and tests are requested?

Reply:
Sure, let's give it a try :)

Reply:
Ah, no good, Alpine does not have this library. But at least Debian now runs this test ;)